includes/Modules/Reader_Revenue_Manager.php

```diff
@@ -240,7 +240,7 @@ protected function setup_info() {
 			'name'        => _x( 'Reader Revenue Manager', 'Service name', 'google-site-kit' ),
 			'description' => __( 'Reader Revenue Manager helps publishers grow, retain, and engage their audiences, creating new revenue opportunities', 'google-site-kit' ),
 			'order'       => 5,
-			'homepage'    => __( 'https://readerrevenue.withgoogle.com/', 'google-site-kit' ),
+			'homepage'    => __( 'https://publishercenter.google.com', 'google-site-kit' ),
```

Comment:
Why is this URL localised? 🤔 

We can't expect (community) localisers to know the right URL to send users to, so let's remove this.

```suggestion
			'homepage'    => 'https://publishercenter.google.com',
```

Reply:
This is actually existent in all modules. Should we maybe create an issue to remove all of them as well?

Reply:
As this is a very small change, I've made this myself. I've also opened [#9175](https://github.com/google/site-kit-wp/issues/9175) to do this for other modules.